src/figuresClasses.ts

```diff
@@ -1,19 +1,81 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
```

Comment:
type Shape = 'triangle' | 'circle' | 'rectangle';
type Color = 'red' | 'green' | 'blue';
```suggestion
  shape: Shape;
  color: Color;
```